apps/i18n/common/en_us.json

```diff
@@ -2377,6 +2380,9 @@
   "updateUnpluggedLessonProgress": "Update unplugged lesson progress",
   "updateUnpluggedLessonProgressSubHeading": "Make sure your report accurately reflects the unplugged lessons* your class has worked on.",
   "updateUnpluggedProgress": "Update unplugged progress",
+  "upgradeAccountToAssign": "Use a teacher account to assign",
```

Comment:
Nit: For consistency in dialog headers, should this line also include something about "assign a curriculum" or "assign curricula" or something?

Reply:
I like that –– going with "Use a teacher account to assign a curriculum"